operate/client/playwright.config.ts

```diff
@@ -26,7 +26,12 @@ const getPort = () => {
 };
 
 const getBasePath = () => {
-  return IS_CI && !IS_E2E ? '' : '/operate/';
+  const isLocalVisualRegression =
+    !IS_CI && !IS_E2E && !IS_A11Y && !IS_SCREENSHOT_GENERATOR;
+
+  const path = (IS_CI && !IS_E2E) || isLocalVisualRegression ? '' : '/operate/';
+
+  return path;
```

Comment:
This fixes the Visual regression tests breaking locally.

There's still improvements we can do later on to make this solution more elegant, but for now this means E2E and Visual regression tests should be working again both on CI and locally when this gets merged.